xilem_core/src/views/mod.rs

```diff
@@ -3,3 +3,13 @@
 
 mod memoize;
 pub use memoize::{memoize, Memoize};
+
+#[path = "one_of.rs"]
+mod one_of_;
```

Comment:
This needs an explanatory comment. I would personally prefer to export this as a normal module.

I can guess that this is so that you don't expose `OneOf2State`, but this isn't very scrutable. Once we enable `unreachable_pub` (which I am planning on doing), this will cause warnings.

Reply:
The alternative is also not that complicated, right? We can have private inline `mod [<private $num_word>] {}`s for those declarations.

Reply:
> so that you don't expose `OneOf2State`

yes that was indeed the motivation, I've done this now via a separate `hidden` module in `one_of.rs`, that is indeed a little bit cleaner, I've also added `#[allow(unreachable_pub)]` with that in warning in mind.